src/components/SelectInput/SelectInput.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import Select from 'react-select';
 import FormLabel from '../FormLabel/FormLabel';
 import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
-import './SelectInput.scss';
+import '../../styles/globals/react-select.scss';
```

Comment:
After further consideration, I don't believe this file belongs in the same dir with utilities. Regardless of its styles being wrapped in `:global`, they are local, and very specific to the component. Since we will never import react-select into another component, this file should probably just be the sole scss file in the component's dir.